src/main/java/com/diamondfire/helpbot/bot/command/impl/other/fun/GarfieldCommand.java

```diff
@@ -40,23 +38,18 @@ public Permission getPermission() {
     @Override
     public void run(CommandEvent event) {
         EmbedBuilder builder = new EmbedBuilder();
-        try {
-            URL url = new URL("https://labscore.vercel.app/v2/garfield");
-            try (BufferedReader in = new BufferedReader(new InputStreamReader(url.openStream()))) {
-                String link = JsonParser.parseString(in.readLine()).getAsJsonObject().get("link").getAsString();
-                
-                if (link == null) {
-                    throw new IOException();
-                } else {
-                    builder.setTitle("Garfield Comic");
-                    builder.setImage(link);
-                    builder.setColor(new Color(252, 166, 28));
-                }
-            }
-        } catch (Exception e) {
-            builder.setTitle(":rotating_light: API BROKE :rotating_light:");
-            builder.setDescription("DM: <@223518178100248576>\nPING: <@223518178100248576>");
-        }
+
+        final Date d1 = new GregorianCalendar(1978, Calendar.JUNE, 19).getTime();
+        final Date d2 = new GregorianCalendar(1999, Calendar.DECEMBER, 31).getTime();
+        Date randomDate = new Date(ThreadLocalRandom.current()
+            .nextLong(d1.getTime(), d2.getTime()));
+
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy/yyyy-MM-dd");
+
+        builder.setTitle("Garfield Comic");
+        builder.setImage(String.format("https://derpystuff.gitlab.io/garf/%s.gif", formatter.format(randomDate)));
```

Comment:
Please store these as static constants

public static final long MIN = new GregorianCalendar(1978, Calendar.JUNE, 19).getTime().getTime();
public static final long MAX = new GregorianCalendar(1999, Calendar.DECEMBER, 31).getTime().getTime();
public static final SimpleDateFormat FORMAT = new SimpleDateFormat("yyyy/yyyy-MM-dd");

Date randomDate = new Date(ThreadLocalRandom.current()
            .nextLong(d1.getTime(), d2.getTime()));
->
Date randomDate = new Date(ThreadLocalRandom.current().nextLong(MIN, MAX));

Reply:
Basically recalculating all these values, the only thing that doesn't need to be a field here is the random date.
Let me know if you have any more questions.